orchagent/mplsrouteorch.cpp

```diff
@@ -121,12 +123,14 @@ void RouteOrch::doLabelTask(Consumer& consumer)
 
             if (op == SET_COMMAND)
             {
+                SWSS_LOG_INFO("Set operation");
```

Comment:
Is this really required? Any route flaps can easily fill the logs without providing any useful information.

Reply:
Removed